setup.py

```diff
@@ -1,14 +1,35 @@
 #!/usr/bin/env python3
-from os.path import isfile
+from os.path import isfile, isdir, join
+from os import listdir, mkdir
+from shutil import copy
 
 from settings import config_path, root_path, save_private_key
 
+
+def get_user_config_choice():
+    options = listdir(configs_source)
```

Comment:
I feel like it makes sense to define this `configs_source` var closer to its usage? it confused me at first!